From 864b7ffd84e62359a166bb9270019900922512e6 Mon Sep 17 00:00:00 2001
From: Pierre-Yves MORDRET <pierre-yves.mordret@foss.st.com>
Date: Wed, 17 May 2023 15:27:11 +0200
Subject: [PATCH 522/529] ARM: dts: stm32: update gpu_reserved memory node on
 stm32mp157[a|d]-ed1

Align gpu_reserved memory node onto stm32mp157c-ed1 and stm32mp157f-ed1
expectation

Change-Id: Icda4e7f3376cd6a51c47628fa4a353b0d03a9c97
Signed-off-by: Pierre-Yves MORDRET <pierre-yves.mordret@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/307012
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
---
 arch/arm/boot/dts/stm32mp157a-ed1.dts | 4 ++--
 arch/arm/boot/dts/stm32mp157d-ed1.dts | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

--- a/arch/arm/boot/dts/stm32mp157a-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157a-ed1.dts
@@ -79,8 +79,8 @@
 			no-map;
 		};
 
-		gpu_reserved: gpu@f6000000 {
-			reg = <0xf6000000 0x8000000>;
+		gpu_reserved: gpu@e8000000 {
+			reg = <0xe8000000 0x8000000>;
 			no-map;
 		};
 	};
--- a/arch/arm/boot/dts/stm32mp157d-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157d-ed1.dts
@@ -79,8 +79,8 @@
 			no-map;
 		};
 
-		gpu_reserved: gpu@f6000000 {
-			reg = <0xf6000000 0x8000000>;
+		gpu_reserved: gpu@e8000000 {
+			reg = <0xe8000000 0x8000000>;
 			no-map;
 		};
 	};
